packages/atomic/cypress/integration/facets/facet-selectors.ts

```diff
@@ -0,0 +1,87 @@
+export const FacetSelectors = {
+  facetStandard: 'atomic-facet',
+  facetSearchbox: 'input[part="search-input"]',
+  checkbox: 'input[type="checkbox"]',
+  showMoreButton: 'button[part="show-more"]',
+  showLessButton: 'button[part="show-less"]',
+  clearAllButton: 'button[part="reset-button"]',
+  numericFacet: 'atomic-numeric-facet',
+  dateFacet: 'atomic-date-facet',
+};
+
+export const BreadcrumbSelectors = {
+  breadcrumb: 'atomic-breadcrumb-manager',
+};
+
+interface IAlias {
+  [key: string]: string;
+}
+
+export const FacetAlias: IAlias = {
+  facetShadow: '@facetShadow',
+  facetUL: '@facet_ul',
+  facetFirstValueLabel: '@facetFirstValueLabel',
+  facetSecondValueLabel: '@facetSecondValueLabel',
+  facetAllValueLabel: '@facetAllValueLabel',
+  facetAllValueCount: '@facetAllValueCount',
+};
+
+const assignedFacetAlias = Object.assign({}, FacetAlias);
+
+Object.keys(assignedFacetAlias).forEach((key: string) => {
```

Comment:
you can add an alias builder/parser in the generic utils, because removing the @ sign will be necessary in every files I guess